components/dot_menu/dot_menu.tsx

```diff
@@ -283,18 +284,22 @@ class DotMenu extends React.PureComponent<Props, State> {
         if (!binding.call) {
             return;
         }
-        const res = await this.props.actions?.doAppCall({
-            ...binding.call,
-            type: AppCallTypes.SUBMIT,
-            context: {
-                app_id: binding.app_id,
-                location: binding.location,
-                team_id: this.props.teamId,
-                channel_id: this.props.post.channel_id,
-                post_id: this.props.post.id,
-                root_id: this.props.post.root_id,
+        const context = createCallContext(
+            binding.app_id,
+            binding.location,
+            this.props.post.channel_id,
+            this.props.teamId,
+            this.props.post.id,
+            this.props.post.root_id,
+        );
+        const call = createCallRequest(
+            binding.call,
+            context,
+            {
+                post: AppExpandLevels.ALL,
             },
-        });
+        );
+        const res = await this.props.actions?.doAppCall(call, AppCallTypes.SUBMIT);
```

Comment:
Why is actions optional?

Reply:
Probably the original proptypes did not have the isrequired, and was just migrated like that.